pyvista/core/filters/image_data.py

```diff
@@ -992,6 +992,7 @@ def contour_labels(  # type: ignore[misc]
         background_value: int = 0,
         select_inputs: int | VectorLike[int] | None = None,
         select_outputs: int | VectorLike[int] | None = None,
+        closed_surface: bool = True,
```

Comment:
Not sure if `closed_surface` is the best name for this. It's possible the contour isn't actually closed even if its sides are. Maybe `capping` is a better term?

Reply:
+1 `pad_background`
```suggestion
        pad_background: bool = True,
```